spec/openapi.yaml

```diff
@@ -1906,11 +1905,10 @@ paths:
                     account_id: my_account,
                     created_date: 2023-08-14T04:21:55.958Z,
                     last_modified_date: 2023-08-14T04:21:55.958Z,
-                    expiry: 2024-08-03T04:21:55.958Z
                   - id: 4a9cb0f4-f383-40b5-84dc-bbb6a3b210dd
                     sender_address: EXAMPLE3
```

Comment:
Please change to `+61491570156`

Reply:
I would change it, is it just to look more explicit?

Reply:
it's because it should be an international phone number, not an alpha tag 👍

Reply:
yeah, agree.

Reply:
updated